lib/screens/settings/settings_view_model.dart

```diff
@@ -104,6 +111,23 @@ abstract class SettingsViewModelBase with Store, ViewModel {
     isCrashManagerEnabled = !isCrashManagerEnabled;
   }
 
+  Row getCrashlyticsRow(BuildContext context) {
+    if (!kIsWeb) {
+      return Row(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Text(LocaleKeys.crash_manager_activated_label.localized()),
+          Switch(
+              activeColor: Theme.of(context).primaryColorDark,
+              value: isCrashManagerEnabled,
+              onChanged: toggleCrashManager),
+        ],
+      );
+    } else {
+      return Row();
+    }
+  }
+
```

Comment:
1) Changed the Function return type to 'Widget'. 
In the else case, return "const SizedBox.shrink()" (More efficient way to render a empty view).